scripts/chemistry/periodic_table/elements.tmcf

```diff
@@ -0,0 +1,8 @@
+
+Node: E:Elements->E0
+typeOf: dcs:ChemicalElement
+name: C:Elements->Element
+chemicalName: C:Elements->Element
+chemicalSymbol: C:Elements->Symbol
+atomicNumber: C:Elements->AtomicNumber
+chemicalSubstanceType: C:Elements->Type
```

Comment:
@ajaits , Don't we need a dcid field in the tmcf?

Reply:
Thanks! 
Yes, adding the dcid: fixed the resolution.